operator/v1/types_descheduler.go

```diff
@@ -0,0 +1,52 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// KubeDescheduler provides information to configure an operator to manage kube-descheduler.
+type KubeDescheduler struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// spec holds user settable values for configuration
+	// +required
+	Spec KubeDeschedulerSpec `json:"spec"`
+	// status holds observed values from the cluster. They may not be overridden.
+	// +optional
+	Status KubeDeschedulerStatus `json:"status"`
+}
+
+// KubeDeschedulerSpec defines the desired state of KubeDescheduler
+type KubeDeschedulerSpec struct {
+	OperatorSpec `json:",inline"`
+
+	// Policy contains the name of an upstream Descheduler policy configmap in the openshift-kube-descheduler-operator project
+	Policy string
```

Comment:
ConfigMapNameReference instead of string, similarly to how we do it in https://github.com/openshift/api/blob/master/config/v1/types_scheduling.go#L30

Reply:
That actually raises a good point, because that is in the scheduler config api in `config/v1` (so the type is `configv1.ConfigMapNameReference`).

Is it worth it to break this descheduler type into 2 apis (operator/config) to use that same type and be more in line with our other operators?

Reply:
> Is it worth it to break this descheduler type into 2 apis (operator/config) to use that same type and be more in line with our other operators?

Yeah, I think it's fine and consistent with other operators.

Reply:
👍 then I'll update this PR to do that